### DISCUSSION TITLE
Change the way "advertising is forbidden" works

### DISCUSSION THREAD

**Discussion by MarcelVos96** [OpenRCT2/OpenRCT2#21877] [Feature requests]
When this option is enabled in vanilla you can still go to the advertising tab and set up an advertisement, but when you actually try to activate it you will get a message saying that advertising is forbidden.

OpenRCT2 recently changed that and now the advertising tab is just entirely gone instead. This was a bit confusing to me initially as I thought it was a bug that it was gone.

Here are three ideas to solve this (2nd and 3rd come from Gymnasiast)
1. Change it back to vanilla.
2. Have the tab there and when you go it show a message saying advertising is disabled instead of available ad campaigns.
3. Have the tab there, but greyed out and disabled, and when you click on it you get the same message as you get in vanilla when you try to activate it.

Idea 3 has my personal preference.

**Comment by MASHzj** [comment]
Option 3 is the best

**Comment by Gymnasiast** [comment]
The advantage of option 3 is also that we can easily do the same to the Land and Water buttons, for parks where that is forbidden. Makes it more consistent, which is more user-friendly.

**Reply by J-PIE-314** [reply]
And if a user tries to click on the grayed out tab, an error message could appear explaining that advertising campaigns are forbidden.

**Reply by AaronVanGeffen** [reply]
We used to have a way to render disabled tabs (#5138). However, that was only ever used by the Twitch tab, so it was removed again when tabs were standardised (#12671). Perhaps worth revisiting?
